src/08-tasks/tasks.apib

```diff
@@ -105,7 +105,11 @@ Update a tasks.
         + description (string, optional)
         + due_at: `2016-02-04T16:44:33+00:00` (string, optional)
         + work_type_id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string, optional)
-        + estimated_duration: 3600 (number, optional) - In seconds
+        + estimated_duration (object, optional)
+            + unit: `s` (enum[string], required)
+                + Members
+                    + s - Seconds
+            + value: `60` (number, required)
```

Comment:
Noob Q: Does apiary allows a way to state that we require a positive number? 🤔

Reply:
you can add comments if necessary